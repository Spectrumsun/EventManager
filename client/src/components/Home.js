import React from 'react';

const style1 = {
  textAlign: 'justify'
};

const style2 = {
  width: '20rem'
};


const home = () => (
  <div>
    <div id="carouselExampleSlidesOnly" className="carousel slide shift" data-ride="carousel">
      <ol className="carousel-indicators">
        <li data-target="#carouselExampleIndicators" data-slide-to="0" className="active" />
        <li data-target="#carouselExampleIndicators" data-slide-to="1" />
        <li data-target="#carouselExampleIndicators" data-slide-to="2" />
      </ol>
      <div className="carousel-inner">
        <div className="carousel-item active">
          <img className="d-block w-100" src="./image/image1.jpg" alt="First slide" />
          <div className="carousel-caption d-none d-md-block caption">
            <h3>Our Event Centers are the best</h3>
            <p>You dont need to look any further</p>
          </div>
        </div>
        <div className="carousel-item">
          <img className="d-block w-100" src="./image/image2.jpg" alt="Second slide" />
          <div className="carousel-caption d-none d-md-block caption">
            <h3>World class center to host your events</h3>
            <p>Very easy to book and check</p>
          </div>
        </div>
        <div className="carousel-item">
          <img className="d-block w-100" src="./image/image3.jpg" alt="Third slide" />
          <div className="carousel-caption d-none d-md-block caption">
            <h3>Chooce from variety of Places</h3>
            <p>Just Login or Signup to access..</p>
          </div>
        </div>
      </div>
    </div>
    <div>
      <div>
        <div className="container">
          <h1 className="center-item shadow" >The Best</h1>
          <p style={style1}>Manage your events easily. Pick from our different centers to host your events. Each center has different facility. projectors, car pack, swimming pools.  We have centers for all types of events you want to organize.
                Easily check if a center is book up while you plan your event. Itâ€™s that easy and simple just create an account and browse through all our center and pick the one you like.
          </p>
          <h1 className="center-item shadow">Current Events</h1>
          <br />
          <div className="card d-lg-inline-block" style={style2}>
            <img className="card-img-top" src="./image/image33.jpg" alt="Card cap" />
            <div className="card-body">
              <h4 className="card-title">Birthday Party</h4>
              <p className="card-text">A event to celebrate Life</p>
              <h6 className="card-text left">12-12-2017 | </h6>
              <h6 className="card-text">10:55 AM</h6>
              <a href="eventinfo.html" className="btn btn-primary">View</a>
            </div>
          </div>
          <div className="card d-lg-inline-block" style={style2}>
            <img className="card-img-top" src="./image/image2.jpg" alt="Card cap" />
            <div className="card-body">
              <h4 className="card-title">Wedding </h4>
              <p className="card-text">A wedding party.</p>
              <h6 className="card-text left">12-12-2017 | </h6>
              <h6 className="card-text">10:55 AM</h6>
              <a href="eventinfo.html" className="btn btn-primary">View</a>
            </div>
          </div>
          <div className="card d-lg-inline-block" style={style2}>
            <img className="card-img-top" src="./image/image3.jpg" alt="Card  cap" />
            <div className="card-body">
              <h4 className="card-title">Seminar</h4>
              <p className="card-text">Having a Growth mindset.</p>
              <h6 className="card-text left">12-12-2017 | </h6>
              <h6 className="card-text">10:55 AM</h6>
              <a href="eventinfo.html" className="btn btn-primary">View</a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
);

export default home;
