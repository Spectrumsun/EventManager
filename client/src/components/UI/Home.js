import React from 'react';
import Centers from '../Center/ViewCenters';

const style1 = {
  textAlign: 'justify'
};


const home = () => (
  <div>
    <div id="carouselExampleSlidesOnly" className="carousel slide shift" data-ride="carousel">
      <ol className="carousel-indicators">
        <li data-target="#carouselExampleIndicators" data-slide-to="0" className="active" />
        <li data-target="#carouselExampleIndicators" data-slide-to="1" />
        <li data-target="#carouselExampleIndicators" data-slide-to="2" />
      </ol>
      <div className="carousel-inner">
        <div className="carousel-item active">
          <img className="d-block w-100" src="https://res.cloudinary.com/skybound/image/upload/v1521281912/image3.jpg" alt="First slide" />
          <div className="carousel-caption d-none d-md-block caption">
            <h3>Our Event Centers are the best</h3>
            <p>You dont need to look any further</p>
          </div>
        </div>
        <div className="carousel-item">
          <img className="d-block w-100" src="https://res.cloudinary.com/skybound/image/upload/v1521281912/image2.jpg" alt="Second slide" />
          <div className="carousel-caption d-none d-md-block caption">
            <h3>World class center to host your events</h3>
            <p>Very easy to book and check</p>
          </div>
        </div>
        <div className="carousel-item">
          <img className="d-block w-100" src="https://res.cloudinary.com/skybound/image/upload/v1521281912/image33.jpg" alt="Third slide" />
          <div className="carousel-caption d-none d-md-block caption">
            <h3>Chooce from variety of Places</h3>
            <p>Just Login or Signup to access..</p>
          </div>
        </div>
      </div>
    </div>
    <div>
      <div>
        <div className="container">
          <h1 className="center-item shadow" >The Best</h1>
          <p style={style1}>Manage your events easily. Pick from our different centers to host your events. Each center has different facility. projectors, car pack, swimming pools.  We have centers for all types of events you want to organize.
                Easily check if a center is book up while you plan your event. Itâ€™s that easy and simple just create an account and browse through all our center and pick the one you like.
          </p>
          <Centers />
        </div>
      </div>
    </div>
    <footer >
      <p>Event Managment &copy; 2017</p>
    </footer>
  </div>

);

export default home;
