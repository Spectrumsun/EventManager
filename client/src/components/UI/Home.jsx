import React from 'react';
import Centers from '../Center/ViewCenters';

/**
 * @description renders component to the DOM
 *
 * @function home
 *
 * @returns {JSX} JSX component for the home
 */
const home = () => (
  <div>
    <div
      id="carouselExampleSlidesOnly"
      className="carousel slide shift"
      data-ride="carousel"
    >
      <ol className="carousel-indicators">
        <li
          data-target="#carouselExampleIndicators"
          data-slide-to="0"
          className="active"
        />
        <li
          data-target="#carouselExampleIndicators"
          data-slide-to="1"
        />
        <li
          data-target="#carouselExampleIndicators"
          data-slide-to="2"
        />
      </ol>
      <div className="carousel-inner">
        <div className="carousel-item active">
          <img
            className="d-block w-100"
            src="https://res.cloudinary.com/skybound/image/upload/v1522444977/eventmanager/static/image3.jpg"
            alt="First slide"
          />
          <div className="carousel-caption d-none d-md-block caption">
            <h3>Our Event Centers are the best</h3>
            <p>You dont need to look any further</p>
          </div>
        </div>
        <div className="carousel-item">
          <img
            className="d-block w-100"
            src="https://res.cloudinary.com/skybound/image/upload/v1522444978/eventmanager/static/image2.jpg"
            alt="Second slide"
          />
          <div className="carousel-caption d-none d-md-block caption">
            <h3>World class center to host your events</h3>
            <p>Very easy to book and check</p>
          </div>
        </div>
        <div className="carousel-item">
          <img
            className="d-block w-100"
            src="https://res.cloudinary.com/skybound/image/upload/v1522444977/eventmanager/static/image33.jpg"
            alt="Third slide"
          />
          <div className="carousel-caption d-none d-md-block caption">
            <h3>Chooce from variety of Places</h3>
            <p>Just Login or Signup to access..</p>
          </div>
        </div>
        <div>
          <div className="list-group-item list-group-item-secondary">
            <h1
              className="center-item shadow"
              style={{ marginTop: '40px' }}
            >The Best
            </h1>
            <p
              className="container read"
              style={{ textAlign: 'justify' }}
            >
                Manage your events easily.
                Pick from our different centers to host your events.
                Each center has different facility. 
                projectors, car pack, swimming pools.
                We have centers for all types
                of events you want to organize.
                Easily check if a center is
                book up while you plan your event.
                Itâ€™s that easy and simple just
                create an account and browse
                through all our center and pick the one you like.
                Lorem ipsum dolor sit amet,
                consetetur sadipscing elitr,
                sed diam nonumy eirmod tempor
                invidunt ut labore et dolore magna aliquyam erat,
                sed diam voluptua. At vero eos et accusam et
                justo duo dolores et ea rebum.
                Stet clita kasd gubergren, no sea takimata
                \sanctus est Lorem ipsum dolor sit amet.
                Lorem ipsum dolor sit amet, consetetur
                sadipscing elitr,sed diam nonumy eirmod tempor
                invidunt ut labore et dolore magna aliquyam erat,
                sed diam voluptua. At vero eos et accusam et justo
                duo dolores et ea rebum. Stet clita kasd gubergren, no
                sea takimata sanctus est Lorem ipsum dolor sit amet.
                Lorem ipsum dolor sit amet, consetetur
                sadipscing elitr, sed diam nonumy eirmod
                tempor invidunt ut labore et dolore magna
                aliquyam erat, sed diam voluptua. At vero
                eos et accusam et justo duo dolores et ea
                rebum. Stet clita kasd gubergren, no sea takimata
                sanctus est Lorem ipsum dolor sit amet.
            </p>
          </div>
        </div>
      </div>
    </div>
    <div >
      <Centers />
    </div>
  </div>

);

export default home;
