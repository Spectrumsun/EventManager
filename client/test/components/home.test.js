import React from 'react';
import { mount } from 'enzyme';
import { Provider } from 'react-redux';
import render from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Home from '../../src/components/UI/Home';

const testRender = render.create(
  <BrowserRouter>
    <Provider>
      <Home />
    </Provider>
  </BrowserRouter>);

describe('<Homepage/>', () => {
  const wrapper = mount(
    <BrowserRouter>
      <Provider>
        <Home />
      </Provider>
    </BrowserRouter>);
  it('renders <Homepage/> component', () => {
    expect(testRender).toMatchSnapshot();
  });
});
