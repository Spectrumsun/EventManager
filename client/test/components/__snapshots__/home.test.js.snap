// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render initial layout of HomePage 1`] = `
Array [
  <div>
    <div
      className="carousel slide shift"
      data-ride="carousel"
      id="carouselExampleSlidesOnly"
    >
      <ol
        className="carousel-indicators"
      >
        <li
          className="active"
          data-slide-to="0"
          data-target="#carouselExampleIndicators"
        />
        <li
          data-slide-to="1"
          data-target="#carouselExampleIndicators"
        />
        <li
          data-slide-to="2"
          data-target="#carouselExampleIndicators"
        />
      </ol>
      <div
        className="carousel-inner"
      >
        <div
          className="carousel-item active"
        >
          <img
            alt="First slide"
            className="d-block w-100"
            src="https://res.cloudinary.com/skybound/image/upload/v1522444977/eventmanager/static/image3.jpg"
          />
          <div
            className="carousel-caption d-none d-md-block caption"
          >
            <h3>
              Our Event Centers are the best
            </h3>
            <p>
              You dont need to look any further
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <img
            alt="Second slide"
            className="d-block w-100"
            src="https://res.cloudinary.com/skybound/image/upload/v1522444978/eventmanager/static/image2.jpg"
          />
          <div
            className="carousel-caption d-none d-md-block caption"
          >
            <h3>
              World class center to host your events
            </h3>
            <p>
              Very easy to book and check
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <img
            alt="Third slide"
            className="d-block w-100"
            src="https://res.cloudinary.com/skybound/image/upload/v1522444977/eventmanager/static/image33.jpg"
          />
          <div
            className="carousel-caption d-none d-md-block caption"
          >
            <h3>
              Chooce from variety of Places
            </h3>
            <p>
              Just Login or Signup to access..
            </p>
          </div>
        </div>
        <div>
          <div
            className="list-group-item list-group-item-secondary"
          >
            <h1
              className="center-item shadow"
              style={
                Object {
                  "marginTop": "40px",
                }
              }
            >
              The Best
            </h1>
            <p
              className="container read"
              style={
                Object {
                  "textAlign": "justify",
                }
              }
            >
              Manage your events easily. Pick from our different centers to host your events. Each center has different facility. projectors, car pack, swimming pools. We have centers for all types of events you want to organize. Easily check if a center is book up while you plan your event. Itâ€™s that easy and simple just create an account and browse through all our center and pick the one you like. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
            </p>
          </div>
        </div>
      </div>
    </div>
    <div>
      <Connect(Centers) />
    </div>
  </div>,
]
`;
